lib/list.js

```diff
@@ -11,9 +11,16 @@ function list (opts) {
   return new Promise(function (resolve, reject) {
     opts = R.clone(opts || {});
     validate(opts);
-    const url = buildUrl(opts);
 
-    request(url, opts.throttle)
+    let options = {
```

Comment:
make this a const, and add a `proxy: opts.proxy` instead of using a separated if. The same applies for the resto of the files.